python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -294,7 +297,14 @@ def fit(self, data, epochs=1, batch_size=32):
             self.trainer = Trainer(logger=False, max_epochs=epochs,
                                    checkpoint_callback=self.checkpoint_callback,
                                    num_processes=self.num_processes, use_ipex=self.use_ipex,
-                                   distributed_backend="spawn")
+                                   weights_summary=None, distributed_backend="subprocess")
+
+            main_process = is_main_process()
```

Comment:
comment this? why this warning won't be shown to users when num_processes = 1 or local_distributed_backend = "subprocess"